packages/webapp/src/containers/Finances/index.jsx

```diff
@@ -98,7 +98,10 @@ const Finances = ({ history }) => {
   };
 
   const makeTransactionsSearchableString = (transaction) =>
-    [transaction.note, transaction.typeLabel || t('SALE.FINANCES.LABOUR_EXPENSE')]
+    [
```

Comment:
This fixes an error from https://github.com/LiteFarmOrg/LiteFarm/pull/2945 where the translation strings were mangled and confused 🙂 Running the `i18n` script uncovered this error.